src/worker-thread/index.amp.ts

```diff
@@ -93,6 +101,12 @@ const globalScope: GlobalScope = {
   SVGElement,
   Text,
   Event: WorkerDOMEvent,
+  MouseEvent: MouseEvent,
+  TouchEvent: TouchEvent,
+  FocusEvent: FocusEvent,
+  KeyboardEvent: KeyboardEvent,
+  WheelEvent: WheelEvent,
+  InputEvent: InputEvent,
```

Comment:
The same linter's errors as in `src/worker-thread/index.ts`

Reply:
done